src/screens/Lessons/screens/Lesson/components/Data/index.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import {map, size} from 'lodash'
+import {
+  titleTitleText,
+  instructorTitleText,
+  technologyTitleText,
+  summaryTitleText,
+  videoTitleText,
+} from 'utils/text'
+import Heading from 'components/Heading'
+import Avatar from 'components/Avatar'
+import WistiaVideo from './components/WistiaVideo'
+
+export default ({lesson}) => {
+
+  const items = [
+    {
+      title: titleTitleText,
+      children: lesson.title,
+    },
+    {
+      title: instructorTitleText,
+      children: (
+        <div className='flex items-center'>
+          <div className='mr2'>
+            <Avatar
+              name={lesson.instructor.first_name}
+              url={lesson.instructor.avatar_url}
+              size={2}
+            />
+          </div>
+          {lesson.instructor.full_name}
```

Comment:
destructure `first_name`, `avatar_url`, and `full_name` from `lesson.instructor` maybe?